bedrock/newsletter/templates/newsletter/includes/form.html

```diff
@@ -8,10 +8,19 @@
 
 {% if not success %}
   <form id="newsletter-form" class="mzp-c-newsletter-form" action="{{ url('newsletter.subscribe') }}" method="post">
-    {{ form.newsletters|safe }}
+    {% if is_multi_newsletter_form %}
+      <div id="newsletter-error-strings" data-form-checkboxes-error="{{ ftl('multi-newsletter-form-checkboxes-error') }}"></div>
+    {% else %}
+      {{ form.newsletters|safe }}
```

Comment:
@maureenlholland I did notice one other regression here (not sure if it's a problem or not?). If someone disables JS and tries to fill out http://localhost:8000/en-US/newsletter/, then it's no longer possible to subscribe. The form fails, but without any kind of error useful message. I'm guessing because it's missing this `newsletters` hidden field.

This might not be a problem if we decide it's OK, but it does effect:
- Older browsers where we no longer serve JS.
- Anyone who disabled JS through an add-on.

Might there be a fallback solution that retains some functionality? (perhaps the checkboxes might work here if they are named differently).

Reply:
Hmm, I wasn't aware of that functionality. I'll look into a fallback.

Reply:
I don’t think this should be a blocker, just wondered if there might be an HTML only solution.

Reply:
If I keep the `name="newsletters"` attribute on checkbox inputs, this returns a list instead of a string in the request.
<img width="314" alt="string" src="https://user-images.githubusercontent.com/19650432/156185569-0bc44aa4-05e8-4ef3-b26c-cd4d44c330fe.png">
<img width="291" alt="list" src="https://user-images.githubusercontent.com/19650432/156185583-f5751c5c-99c6-45c7-9b70-94a04a4e2e57.png">

@robhudson @pmac I want to check in on the intention of moving basket interaction to the client side https://github.com/mozilla/bedrock/issues/11149

Does this mean we will no longer support submissions with older browsers or disabled JS? (In which case we need to provide an error message). I think Rob looked into supporting CSV newsletters from the NewsletterFooterForm. Would that allow the submission to work directly to Basket without JS?

Reply:
I think we should look at requiring JS for these forms. I'm guessing that the intersection of people with JS disabled on our site and people who want marketing newsletters is very small. That said we could support that on the basket side along with an allow list of return URLs that basket could send the user back to via a redirect for a traditional submission if JS failed.

Reply:
But we're not doing that for this issue/PR.

Reply:
If we no longer want to keep support for non-JS form submissions, then we could look to either add an error message / warning that the forms require JS, or alternatively hide them entirely in browsers where we don't ship JS (or people have it disabled). That needn't have to be done in this PR though.

Reply:
I did add a check that `newsletters` was valid and produce an error if not. But since the end user can't control the hidden field there isn't anything they can do there. So that may be a change from previous behavior. But without that change it would not report any error and subscribe you to the default newsletter.

It is possible that in Python if `newsletters` is a list, that we join it ourselves server-side. That should be a simple change.

Reply:
https://github.com/mozilla/bedrock/issues/11270